tools/localizations/cn.csv

```diff
@@ -35,4 +35,5 @@ Zigbee Switch,Zigbee开关
 Zigbee Thermostat,Zigbee温控
 Zigbee Thing,Zigbee设备
 Zigbee Vent,Zigbee新风系统
-Zigbee Window Treatment,Zigbee窗帘电机
\ No newline at end of file
+Zigbee Window Treatment,Zigbee窗帘电机
+Aqara Smart Smoke Detector,Aqara烟雾报警器
```

Comment:
I've added this localization already: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/main/tools/localizations/cn.csv#L54

Reply:
> I've added this localization already: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/main/tools/localizations/cn.csv#L54

@Hongming6 
Please rebase your PR.